pkg/controller/common/metrics.go

```diff
@@ -0,0 +1,64 @@
+package common
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/golang/glog"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+)
+
+var (
+	// DefaultBindAddress is the port for the metrics listener
+	DefaultBindAddress = ":8797"
```

Comment:
`const` instead of `var`?

Reply:
yes, we don't write to it, but in my defense I stole that verbatim from the daemon. :smiley: 

This is probably in line with Zack's comments about the boilerplate: https://github.com/openshift/machine-config-operator/pull/2802/#discussion_r765177360 whatever we fix here, we should fix across all our metrics listeners so there is "one true metrics listener" setup function.